cluster/store/store.go

```diff
@@ -113,7 +113,8 @@ type Config struct {
 	// LoadLegacySchema is responsible for loading old schema from boltDB
 	LoadLegacySchema LoadLegacySchema
 	// IsLocalHost only required when running Weaviate from the console in localhost
-	IsLocalHost bool
+	IsLocalHost            bool
+	IgnoreWaitingForLeader bool
```

Comment:
The goal is to facilitate the migration from non-raft to raft cluster.
How about renaming it to "MigrateFromNonRaft" or similar?

Reply:
i think that doesn't describe what this flag does then, but instead what we want to achieve from that flag

Reply:
Here, we are essentially faking the readiness probe so for the cluster manager (CM)  to start other nodes. However, the node won’t be ready until all voters are up, which might take some time. It is important to note that this a temporary workaround, not a permanent solution.

Reply:
we came to the point we will revisit that flag later on